Core/src/org/sleuthkit/autopsy/centralrepository/ingestmodule/IngestModule.java

```diff
@@ -129,7 +131,9 @@ public ProcessResult process(AbstractFile abstractFile) {
          */
         if (abstractFile.getKnown() != TskData.FileKnown.KNOWN && flagTaggedNotableItems) {
             try {
+                TimingMetric timingMetric = EnterpriseHealthMonitor.getTimingMetric("Correlation Engine: Notable artifact query");
```

Comment:
Same goes for "Correlation Engine: Notable artifact query" string, I think it should be a constant.